src/assets/style/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+//Mobile-first Media Queries
+
+@mixin media-breakpoint-tablet {
+    @media (min-width: media-breakpoint-tablet) {
```

Comment:
variable name should start with $. Also, can you import this in index.scss and also verify that mixin in working by importing it in hero.scss